### PR TITLE
NEW REG test cases using new IDs: 8

### PR DESCRIPTION
**Pull request by jcereced**
Reg test with new id #8

### CODE REVIEW

File: src/harness/testcases/WINNF_FT_S_REG_testcase.py [hunk 1]
```diff
@@ -423,7 +423,49 @@ def test_WINNF_FT_S_REG_5(self):
...
+
+    # Inject FCC IDs
+    self._sas_admin.InjectFccId({'fccId': device_1['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_2['fccId']})
+    self._sas_admin.InjectFccId({'fccId': device_3['fccId']})
 
+    # Inject User IDs
+    self._sas_admin.InjectUserId({'userId': device_1['userId']})
+    self._sas_admin.InjectUserId({'userId': device_2['userId']})
+    self._sas_admin.InjectUserId({'userId': device_3['userId']})
```
Comment (RedaHossino) [inline src/harness/testcases/WINNF_FT_S_REG_testcase.py]: it looks good, only one optional tiny comment, isn't it shorter to inject User Id and Fcc Id by a loop through the devices?

File: src/harness/testcases/WINNF_FT_S_REG_testcase.py [hunk 2]
```diff
@@ -423,7 +423,49 @@ def test_WINNF_FT_S_REG_5(self):
...
+    # Device 3 out-of-range, or the wrong Type value for latitude.
+    device_3['installationParam']['latitude'] = 91.0
+
+    # Register the devices
+    request = {'registrationRequest': [device_1, device_2, device_3]}
+    response = self._sas.Registration(request)['registrationResponse']
+
+    # Check registration response
+    # valid cbsdId and responseCode 0 for 1st cbsd
+    self.assertTrue('cbsdId' in response[0])
```
Comment (RedaHossino) [inline src/harness/testcases/WINNF_FT_S_REG_testcase.py]: following a discussion with Kate in an email I sent to dev test list(Question about "Validation" of CbsdId), we may need to check the length in each test that asks us to verify that a valid CBSD ID is returned.
do you agree with that?

File: src/harness/testcases/WINNF_FT_S_REG_testcase.py (hunk 1 above)
Comment (jcereced) [reply src/harness/testcases/WINNF_FT_S_REG_testcase.py]: probably....I think we were just modeling things after the way earlier TCs were impemented

Comment (RedaHossino) [reply src/harness/testcases/WINNF_FT_S_REG_testcase.py]: yes, no problem then :)

File: src/harness/testcases/WINNF_FT_S_REG_testcase.py (hunk 2 above)
Comment (RedaHossino) [reply src/harness/testcases/WINNF_FT_S_REG_testcase.py]: I agree with Kate, maybe we cann't do more than checking the length of CBSD ID.

**Comment by jcereced** [comment]
this PR is obsolete,,,created a new one to merge and avoid the conflicts...closing

**Pull request closed without merging**